src/devices/Ws28xx/README.md

```diff
@@ -2,11 +2,12 @@
 
 This binding allows you to update the RGB LEDs on Ws28xx / SK6812 and based strips and matrices.
 
-To see how to use the binding in code, see the [sample](samples/Ws28xx_Samples/Program.cs).
+To see how to use the binding in code, see the [sample](samples/LEDStripSample/Program.cs).
 
 ## Documentation
 
 * WS2812B: [Datasheet](https://cdn-shop.adafruit.com/datasheets/WS2812B.pdf)
+* WS2815B: [Datasheet](http://www.world-semi.com/DownLoadFile/138)
```

Comment:
Please also add very tiny sample here how to use or initialize this class, even a single comment after line 49 will suffice

Reply:
OK.